test(ivy): update root causes for material menu

crisbeto opened the pull request:
Updates the root causes for the remaining `MatMenu` tests. Note that two tests are omitted from this PR, because one will be fixed by https://github.com/angular/angular/pull/28951 and the other one depends on static queries and will be refactored by https://github.com/angular/material2/pull/15289.

kara commented on tools/material-ci/angular_material_test_blocklist.js:
```diff
@@ -763,23 +763,23 @@ window.testBlocklist = {
   "MatMenu animations should enable ripples on items by default": {
     "error": "TypeError: Cannot read property 'query' of null",
-    "notes": "Unknown"
+    "notes": "FW-1059: Diagnose root causes of failures in MatMenu"
```
I think you pasted the wrong summary? Seems like it should be:

```suggestion
    "notes": "FW-1059: DebugNode.query should query nodes in the logical tree"
```

crisbeto replied:
Ops, not sure how that happened. Fixed.

crisbeto committed:
test(ivy): update root causes for material menu

Updates the root causes for the remaining `MatMenu` tests. Note that two tests are omitted from this PR, because one will be fixed by https://github.com/angular/angular/pull/28951 and the other one depends on static queries and will be refactored by https://github.com/angular/material2/pull/15289.

kara reviewed (approved):
LGTM

kara commented:
merge-assistance: core approval should be sufficient for this change once it goes green
